tests/replication/_base.py

```diff
@@ -70,8 +70,16 @@ def prepare(self, reactor, clock, hs):
         # databases objects are the same.
         self.worker_hs.get_datastore().db_pool = hs.get_datastore().db_pool
 
+        # Normally we'd pass in the handler to setup_test_homeserver, which would
```

Comment:
turbo nit: can we wrap `setup_test_homeserver` in backticks, since `cache_in_self` gets that treatment later on in the comment?